src/components/ColumnResizer/ColumnResizer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+interface Props {
+  onDoubleClick?: () => void
+  setResizeWidth?: (width?: number) => void
+  setFinalWidth?: (width: number) => void
+  width?: number
+}
+
+export default function ColumnResizer({ onDoubleClick, setFinalWidth, setResizeWidth, width }: Props) {
```

Comment:
I found it hard to find where to split up the table / header. Nice job splitting out the resizer